2023/11/29 00:33:47 failed to dial tcp connection, retrying
2023/11/29 00:33:48 failed to dial tcp connection, retrying
[2023-11-29T00:33:48.623Z] info  code-server 4.11.0 85e083580dec27ef19827ff42d3c9257d56ea7e3
[2023-11-29T00:33:48.624Z] info  Using user-data-dir ~/.local/share/code-server
[2023-11-29T00:33:48.638Z] info  Using config file ~/.config/code-server/config.yaml
[2023-11-29T00:33:48.638Z] info  HTTP server listening on http://127.0.0.1:8080/
[2023-11-29T00:33:48.638Z] info    - Authentication is disabled
[2023-11-29T00:33:48.638Z] info    - Not serving HTTPS
====================================>token: a447bad7-2dd6-4300-8121-983e3d04c94d
[00:33:51] 




[00:33:51] Extension host agent started.
[00:33:53] [127.0.0.1][c5c892a9][ManagementConnection] New connection established.
[00:33:53] [127.0.0.1][4c585498][ExtensionHostConnection] New connection established.
[00:33:53] [127.0.0.1][4c585498][ExtensionHostConnection] <377> Launched Extension Host Process.
[00:33:57] Using the in-memory credential store as the operating system's credential store could not be accessed. Please see https://aka.ms/vscode-server-keyring on how to set this up. Details: libsecret-1.so.0: cannot open shared object file: No such file or directory
[00:34:07] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /opt/conda/lib/python3.8)
[2023-11-29T00:36:03.894Z] error Failed to get latest version [38;2;140;140;140m{"error":"connect ETIMEDOUT 20.205.243.168:443"}[0m
2023/11/29 02:32:08 exit program after timeout
